lms/djangoapps/courseware/tests/test_module_render.py

```diff
@@ -1589,7 +1589,7 @@ def test_course_image(self):
         self.course.static_asset_path = ""
 
     @override_settings(DEFAULT_COURSE_ABOUT_IMAGE_URL='test.png')
-    @override_settings(STATIC_URL='static/')
+    @override_settings(STATIC_URL='/static/')
```

Comment:
In `Django 3.0+`, `STATIC_URL` is automatically prepended with `/` if it is not present at the start of the relative path. See details: https://docs.djangoproject.com/en/3.1/ref/settings/#std:setting-STATIC_URL.